arch_build.cmake

```diff
@@ -0,0 +1,5 @@
+generate_config_includefile(
+    FILE_NAME           "device_cfg.h" 
+    MATCH_SUFFIX        "device_cfg.h"
+    PREFIXES            ${BUILD_CONFIG}
+)
```

Comment:
I'm unsure why the cFS CMake requires this, but I'm glad you had figured it out previously @mgrubb-gsfc